package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "autoprefixer-loader",
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
Please don't bump the package version number in a PR. In 99% of the cases this doesn't help the maintainer, but makes it harder to publish a new version.